addon/helpers/list.js

```diff
@@ -1,6 +1,16 @@
 import { MicroState } from 'ember-microstates';
+import assign from '../utils/assign';
 
 export default MicroState.extend({
+
+  wrap(value) {
+    return assign(value, {
```

Comment:
I couldn't use `Object.create` because it turns it into an object with prototype array. I used `Array.from` with `Object.defineProperty` instead.